training/code-of-conduct/training-privacy.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: section
+title: ARCHER2 Training Privacy Statement
+banner: web_banners_05.jpg
+---
+
+
+The information you provide will be used by EPCC to administer your attendance at our training courses. The information will be used to send you information about the course and will be retained beyond the end of the course so that we can send you follow-up surveys.
+
+We are using this information about you because it is necessary for the performance of the contracts that provide you with these training courses.
+
+Information about you will be shared with:
+
+- The organisations that host the training courses that you attend. This is necessary to allow them to ensure the courses run smoothly.
+- External bodies (such as UKRI, EPSRC, Software Carpentry, PRACE, ...) that fund or support the courses you attend. This is necessary to justify their funding or support for these courses.
```

Comment:
[pedantry mode on] 😄 
Can we change this list to "... (such as UKRI, The Carpentries, PRACE)..."
[pedantry mode off]